src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -79,7 +79,15 @@ public PreviewControl(NodeViewModel nodeViewModel)
         {
             this.scheduler = nodeViewModel.DynamoViewModel.Model.Scheduler;
             this.nodeViewModel = nodeViewModel;
-            InitializeComponent();
+            try
+            {
+                InitializeComponent();
+            }
+            catch(System.Windows.Markup.XamlParseException e)
+            {
+                // See https://docs.microsoft.com/en-us/dotnet/api/system.windows.markup.xamlparseexception?view=windowsdesktop-6.0#remarks
+                Logging.Analytics.TrackException(e, false);
+            }
```

Comment:
so we want to continue when this exception is caught ?

Reply:
Yes, I think that's what we discussed at standup. Best case scenario, Dynamo may continue to work normally, worst case scenario, node previews will be broken, Dynamo will work otherwise, and it won't crash.

Reply:
Hopefully we do not cause another crash by continuing here

Reply:
I just tested this scenario...and Dynamo crashes immediately afterwards

Reply:
Looks like there is no benefit in handling the exception here.

Reply:
how did you test this? You said you weren't able to reproduce it in https://jira.autodesk.com/browse/DYN-3945